docs/quickstart.rst

```diff
@@ -0,0 +1,601 @@
+.. eql:section-intro-page:: rocket
+
+==========
+Quickstart
+==========
+
+Welcome to EdgeDB!
+
+This quickstart will walk you through the entire process of creating a simple EdgeDB-powered application: installation, defining your schema, adding some data, and writing your first query. Let's jump in!
+
+
+..    :ref:`Installation <ref_quickstart_install>`
+..    :ref:`Initialize a project <ref_quickstart_createdb>`
+..    :ref:`Set up your schema <ref_quickstart_createdb_sdl>`
+..    :ref:`Insert data <ref_quickstart_insert_data>`
+..    :ref:`Run some queries <ref_quickstart_queries>`
+..    :ref:`Migrate your schema <ref_quickstart_migrations>`
+..    :ref:`Computables <ref_quickstart_computables>`
+..    :ref:`Onwards and upwards <ref_quickstart_onwards>`
+
+.. _ref_quickstart_install:
+
+1. Installation
+===============
+
+First let's install the EdgeDB CLI. Open a terminal and run the appropriate command below.
+
+**macOS/Linux**
+
+.. code-block:: bash
+
+    $ curl https://sh.edgedb.com --proto '=https' -sSf1 | sh
+
+**Windows**
+
+.. code-block::
+
+    # in Powershell
+    PS> iwr https://ps1.edgedb.com -useb | iex
+
+This command downloads and executes a bash script that installs the ``edgedb`` CLI on your machine. You may be asked for your password. Once the installation completes, restart your terminal so the ``edgedb`` command becomes available.
+
+Now let's use the CLI to install the latest version of the EdgeDB database. (The CLI is separate from the database itself.)
+
+.. code-block:: bash
+
+    $ edgedb server install
+
+
+> Depending on your OS, the native server packages might not yet be available, and you might need to install and run Docker to complete the EdgeDB server installation.
+
+Now let's set up your EdgeDB project.
+
+.. _ref_quickstart_createdb:
+
+2. Initialize a project
+=======================
+
+In a terminal, create a new directory and ``cd`` into it.
+
+.. code-block:: bash
+
+    $ cd ~/Documents
+    $ mkdir edgedb-quickstart
+    $ cd edgedb-quickstart
+
+Then initialize your EdgeDB project:
+
+.. code-block:: bash
+
+    $ edgedb project init
+
+This starts an interactive tool that walks you through the process of setting up your first EdgeDB instance. You should see something like this:
+
+.. code-block::
+
+    $ edgedb project init
+
+    No `edgedb.toml` found in `~/Documents/edgedb-quickstart`
```

Comment:
`~/Documents/` is  macOS path. How about we just use `~/edgedb-quickstart` which is more cross-platform? (I'm also not sure what to do about Windows)

Reply:
Changed this `~/Desktop` everywhere and added a note indicating that the actual path is customizable:

```
.. code-block:: bash

    $ cd ~/Desktop # <- or wherever you want the project to live
    $ mkdir edgedb-quickstart
    $ cd edgedb-quickstart
```